nixos/modules/services/networking/shellhub-agent.nix

```diff
@@ -1,31 +1,39 @@
 { config, lib, pkgs, ... }:
 
 with lib;
+
 let
   cfg = config.services.shellhub-agent;
-in {
-
+in
+{
   ###### interface
 
   options = {
 
     services.shellhub-agent = {
 
-      enable = mkOption {
-        type = types.bool;
-        default = false;
+      enable = mkEnableOption ''
+        Whether to enable the ShellHub Agent daemon, which allows
+        secure remote logins.
```

Comment:
`mkEnableOption` already adds the "Whether to enable" prefix: https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L97.

Reply:
thanks; fixed.